### PR TITLE
[REV] Modify the oauth style

### PR DESCRIPTION
**Pull request by zsl1549**

### CODE REVIEW

File: src/components/OauthForm/index.js [hunk 1]
```diff
@@ -243,10 +263,17 @@ class CreateOAuthForm extends PureComponent {
...
           >
             {getFieldDecorator("redirect_domain", {
-              initialValue: oauthInfo ? oauthInfo.redirect_uri.replace("/console/oauth/redirect","") : `${window.location.protocol}//${window.location.host}`,
-              rules: [{ required: true, message: "请输入正确的平台访问域名" }, { required: "url", message: "输入数据不是合法的URL" }]
+              initialValue: oauthInfo
+                ? oauthInfo.redirect_uri.replace("/console/oauth/redirect", "")
+                : `${window.location.protocol}//${window.location.host}`,
+              rules: [
+                { required: true, message: "请输入正确的平台访问域名" },
+                { required: "url", message: "输入数据不是合法的URL" }
```
Comment (barnettZQG) [inline src/components/OauthForm/index.js]: type: "url"

File: src/components/OauthForm/index.js [hunk 2]
```diff
@@ -175,13 +189,16 @@ class CreateOAuthForm extends PureComponent {
             >
               {getFieldDecorator("api_url", {
                 initialValue: oauthInfo ? oauthInfo.api_url : "",
-                rules: [{ required: true, message: "请输入获取用户数据的API地址" }, { required: "url", message: "输入数据不是合法的URL" }]
+                rules: [
+                  { required: true, message: "请输入获取用户数据的API地址" },
+                  { required: "url", message: "输入数据不是合法的URL" }
```
Comment (barnettZQG) [inline src/components/OauthForm/index.js]: type: "url"

File: src/components/OauthForm/index.js [hunk 3]
```diff
@@ -156,7 +167,10 @@ class CreateOAuthForm extends PureComponent {
             >
               {getFieldDecorator("access_token_url", {
                 initialValue: oauthInfo ? oauthInfo.access_token_url : "",
-                rules: [{ required: true, message: "请输入access_token_url" }, { required: "url", message: "输入数据不是合法的URL" }]
+                rules: [
+                  { required: true, message: "请输入access_token_url" },
+                  { required: "url", message: "输入数据不是合法的URL" }
```
Comment (barnettZQG) [inline src/components/OauthForm/index.js]: type: "url"

File: src/components/OauthForm/index.js [hunk 4]
```diff
@@ -138,7 +146,10 @@ class CreateOAuthForm extends PureComponent {
             >
               {getFieldDecorator("auth_url", {
                 initialValue: oauthInfo ? oauthInfo.auth_url : "",
-                rules: [{ required: true, message: "请输入认证地址" }, { required: "url", message: "输入数据不是合法的URL" }]
+                rules: [
+                  { required: true, message: "请输入认证地址" },
+                  { required: "url", message: "输入数据不是合法的URL" }
```
Comment (barnettZQG) [inline src/components/OauthForm/index.js]: type: "url"

File: src/components/OauthForm/index.js (hunk 2 above)
Comment (zsl1549) [reply src/components/OauthForm/index.js]: Repair complete

File: src/components/OauthForm/index.js (hunk 1 above)
Comment (zsl1549) [reply src/components/OauthForm/index.js]: Repair complete

File: src/components/OauthForm/index.js (hunk 3 above)
Comment (zsl1549) [reply src/components/OauthForm/index.js]: Repair complete

File: src/components/OauthForm/index.js (hunk 4 above)
Comment (zsl1549) [reply src/components/OauthForm/index.js]: Repair complete

**Pull request merged**